main.php

```diff
@@ -71,6 +70,21 @@
 		exit(1);
 	}
 }
+// Allow a first class env var to be set that disables .env file loading
+if (!getenv('SS_IGNORE_DOT_ENV')) {
```

Comment:
Agree, `null` vs `false` is worse than what we have at the moment, didn't consider the execution order.